src/TodoList.tsx

```diff
@@ -55,14 +55,25 @@ const TodoList = () => {
     )
   }
 
+  const handleClickUpdateButton = (id: number, title: string) => {
+    setTask((prev) =>
+      prev.map((task) => ({
+        date: task.date,
+        todo: task.todo.map((item) =>
+          item.id === id ? { ...item, title: title } : item
+        ),
+      }))
```

Comment:
minor) 구조분해할당을 사용하면 더 간결하게 작성할 수 있어요~

```suggestion
      prev.map(({ date, todo }) => ({
        date,
        todo: todo.map((item) =>
          item.id === id ? { ...item, title: title } : item
        ),
      }))
```